GNUmakefile

```diff
@@ -460,8 +460,14 @@ src/afl-forkserver.o : $(COMM_HDR) src/afl-forkserver.c include/forkserver.h
 src/afl-sharedmem.o : $(COMM_HDR) src/afl-sharedmem.c include/sharedmem.h
 	$(CC) $(CFLAGS) $(CFLAGS_FLTO) -c src/afl-sharedmem.c -o src/afl-sharedmem.o
 
-afl-fuzz: $(COMM_HDR) include/afl-fuzz.h $(AFL_FUZZ_FILES) src/afl-common.o src/afl-sharedmem.o src/afl-forkserver.o src/afl-performance.o | test_x86
-	$(CC) $(CFLAGS) $(COMPILE_STATIC) $(CFLAGS_FLTO) $(AFL_FUZZ_FILES) src/afl-common.o src/afl-sharedmem.o src/afl-forkserver.o src/afl-performance.o -o $@ $(PYFLAGS) $(LDFLAGS) -lm
+src/afl-skipdet.o : $(COMM_HDR) src/afl-skipdet.c
+	$(CC) $(CFLAGS) $(CFLAGS_FLTO) -c src/afl-skipdet.c -o src/afl-skipdet.o
+
+src/afl-skip-log.o : $(COMM_HDR) src/afl-skip-log.cc 
```

Comment:
please remove the .o entries and pass it to the afl-fuzz lines as .c files. we only what to create .o files for those that are used by multiple tools, e.g. also afl-showmap.